io/net/virt-net/network_virtualization.py

```diff
@@ -79,16 +79,26 @@ def setUp(self):
             self.cancel("HMC IP not got")
         self.hmc_pwd = self.params.get("hmc_pwd", '*', default=None)
         self.hmc_username = self.params.get("hmc_username", '*', default=None)
-        self.lpar = self.params.get("lpar", '*', default=None)
+        self.lpar = self.get_mcp_component("NodeNameList").split('.')[0]
+        if not self.lpar:
+            self.cancel("LPAR Name not got from lsrsrc command")
         self.login(self.hmc_ip, self.hmc_username, self.hmc_pwd)
         cmd = 'lssyscfg -r sys  -F name'
         output = self.run_command(cmd)
         self.server = ''
         for line in output:
             if line in self.lpar:
                 self.server = line
+                break
         if not self.server:
             self.cancel("Managed System not got")
+        cmd = 'lssyscfg -r lpar -F name -m %s' % self.server
+        output = self.run_command(cmd)
+        for line in output:
+            if "%s-" % self.lpar in line:
+                self.lpar = line
+        if not self.is_lpar_in_server(self.server, self.lpar):
```

Comment:
We are finding the LPAR from server, in line 95.
So, this is unnecessary.

Reply:
it is needed only to check if lpar is not migrated already ?

Reply:
yes, mainly in LPM context. not necessary here.